content/en/docs/about/membership.md

```diff
@@ -0,0 +1,193 @@
+
+## Reviewer
+
+Reviewers are able to review code for quality and correctness on some part of a subproject. They are knowledgeable about both the codebase and software engineering principles.
+
+**Defined by:** *reviewers* entry in an OWNERS file in a repo owned by the Kubeflow project.
+
+Reviewer status can be scoped to either parts of the codebase or the root directory for the entire codebase
+
+**Note:** Acceptance of code contributions requires at least one approver in addition to the assigned reviewers.
+
+### Requirements
+
+The following apply to the part of codebase for which one would be a reviewer in an [OWNERS] file.
+
+- member for at least 3 months
+- Primary reviewer for at least 5 PRs to the codebase
+- Reviewed or merged at least 20 substantial PRs to the codebase
+- Knowledgeable about the codebase
+- Sponsored by a subproject approver
+  - With no objections from other approvers
+  - Done through PR to update the OWNERS file
+- May either self-nominate or be nominated by an approver in this subproject
+
+**Note:** Working Group Leads may nominate and approve `Reviewers` that don't meet these requirements due to exceptional circumstances. While acceptable in the short term, Working Group Leads should ensure that these `Reviewers` eventually meet the requirements
+
+The following apply to the part of codebase for which one would be a reviewer in an [OWNERS] file (for repos using the bot).
+
+### Responsibilities
+
+- All responsiblities that community members have
+- Responsible for project quality control via [code reviews]
+  - Focus on code quality and correctness, including testing and factoring
+  - May also review for more holistic issues, but not a requirement
+- Expected to be responsive to review requests
+- Assigned PRs to review related to subproject of expertise
+- Assigned test bugs related to subproject of expertise
+
+### Privileges
+
+- All Privileges that community members have
+- Code reviewer status may be a precondition to accepting large code contributions
+- May get a badge on PR and issue comments
+
+## Approver
+
+Code approvers are able to both review and approve code contributions. While
+code review is focused on code quality and correctness, approval is focused on
+holistic acceptance of a contribution including: backwards / forwards
+compatibility, adhering to API and flag conventions, subtle performance and
+correctness issues, interactions with other parts of the system, etc.
+
+**Defined by:** *approvers* entry in an OWNERS file in a repo owned by the Kubeflow project.
+
+Approver status is scoped to a part of the codebase.
+
+### Requirements
+
+The following apply to the part of codebase for which one would be an approver in an [OWNERS] file (for repos using the bot).
+
+- Reviewer of the codebase for at least 3 months
+- Primary reviewer for at least 10 substantial PRs to the codebase
+- Reviewed or merged at least 30 PRs to the codebase
+- Nominated by a WG Lead, Chair or owner
```

Comment:
I would only leave WG Lead or Chair.

Reply:
I was thinking of situations where a root approver may nominate an approver for a subdirectory or similar. WG Leads and Chairs aren't necessarily involved with the day-to-day of every repository right?

Reply:
In that case, we should explain what is root approver and sub-directory approver in terms of control of the projects.
As you can see, WG Leads should be involved into day-to-day actives for every repository: https://github.com/kubeflow/community/blob/master/wgs/wg-governance.md#all-leads 

That is why, I feel that for now we should delegate it to WG Leads or Chairs for simplicity.

Reply:
Any thoughts @anishasthana @terrytangyuan ?